src/index.js

```diff
@@ -0,0 +1,161 @@
+const state = {
+    currentTempButton: null,
+    increaseTempControl: null,
+    decreaseTempControl: null,
+    landscapeDiv: null,
+    tempValue: null,
+    tempCount: 0,
+    skySelection: null,
+    skyDiv: null,
+    cityNameInput: null,
+    cityNameHeader: null,
+    resetButton: null,
+};
+
+const loadControls = () => {
+    // Retrieve references to all HTML elements we need to access
+    state.increaseTempControl = document.getElementById("increaseTempControl");
+    state.decreaseTempControl = document.getElementById("decreaseTempControl");
+    state.tempValue = document.getElementById("tempValue");
+    state.tempValue.textContent = state.tempCount
+    state.currentTempButton = document.getElementById("currentTempButton");
+    state.skySelection = document.getElementById("skySelect");
+    state.skyDiv = document.getElementById("sky");
+    state.landscapeDiv = document.getElementById("landscape");
+    state.cityNameInput = document.getElementById('cityNameInput');
+    state.cityNameHeader = document.getElementById("headerCityName");
+    state.resetButton = document.getElementById("cityNameReset");
+};
+
+const registerEvents = (event) => {
+    state.increaseTempControl.addEventListener("click", handleIncreaseTempClick);
+    state.increaseTempControl.addEventListener("click", displayLandscape);
```

Comment:
For these increase/decrease temp control event handlers, instead of attaching multiple click listeners, it's better to attach one listener and perform multiple things within it. As you saw, attaching multiple event listeners works to accomplish the task in this case but as an application grows there is more likelihood of colliding with the same event, potentially causing an incorrect action/response or even blocking the UI for longer since each listener is queued independently. Putting everything in the same event handler also makes it easier to find out what is happening in your application, in case you need to debug.